src/test/java/com/cloudbees/hudson/plugins/folder/FolderTest.java

```diff
@@ -425,19 +425,19 @@ private void copyViaHttp(Folder f, JenkinsRule.WebClient wc, String fromName, St
 
     @Issue("JENKINS-58282")
     @Test public void shouldHaveHealthMetricConfiguredGloballyOnCreation() throws Exception {
-        assertThat("by default, global configuration should have all folder health metrics",
-                AbstractFolderConfiguration.get().getHealthMetrics(), hasSize((int) FolderHealthMetricDescriptor.all().stream().filter(d -> d.createDefault() != null).count()));
+        assertThat("by default, global configuration should not have any health metrics",
+                AbstractFolderConfiguration.get().getHealthMetrics(), hasSize(0));
```

Comment:
Because now the global configuration shouldn't have any health metrics by default, rename the test from `shouldHaveHealthMetricConfiguredGloballyOnCreation` to `shouldNotHaveHealthMetricConfiguredGloballyOnCreation`.